entry_types/scrolled/package/spec/frontend/usePhonePlatform/inFrontend-spec.js

```diff
@@ -0,0 +1,22 @@
+
+import {PhonePlatformProvider} from 'frontend';
+import {usePhonePlatform} from 'frontend/usePhonePlatform';
+
+import {browser} from 'pageflow/frontend';
+import {renderHook,} from '@testing-library/react-hooks';
+
+import '@testing-library/jest-dom/extend-expect'
+
+
+describe('usePhonePlatform', () => {
+
+  it('returns true on phone platform', () => {
+
+    jest.spyOn(browser, 'has').mockReturnValue(true);
+
+    const {result} = renderHook(() => usePhonePlatform(), {wrapper: PhonePlatformProvider});
+
+    expect(result.current).toEqual(true);
+  });
+
+});
```

Comment:
**suggestion**: Always test both cases for a boolean function.

Both tests for `usePhonePlatform` would also pass if the implementation of the hook was just `return true`.